### ISSUE TITLE
O(1) space, not O(n)

### ISSUE THREAD

**Issue by eliork** [neetcode-gh/leetcode#2465]
The Map will be in the size of the amount of letters, which is 26, not the length of the string

https://github.com/neetcode-gh/leetcode/blob/b98ee992d3d318dc49b3b7a4acd4eb001c6e60e9/javascript/0242-valid-anagram.js#L43

**Comment by a93a** [comment]
Indeed, you are correct. You can do a PR with changes if you would like to contribute, or I can do it otherwise 😄

**Comment by eliork** [comment]
Let me try :-)
